packages/defaults.yaml

```diff
@@ -21,3 +21,13 @@ packages:
     required:
       states: []
       pkgs: []
+  snaps:
+    package: snapd
+    collides: []
+    symlink: False
+    service: ['snapd.service', 'snapd.socket',]
```

Comment:
Mind you just change this line to

```yaml
service: ['snapd']
```
so the salt uses whatever the OS has (systemd, upstart, etc) and that's it? I checked and `snapd.socket` is a dependency of `snapd.service`, so there's no need to include it.

Reply:
thanks for suggestion, fixing that now.